### ISSUE TITLE
wlstreamer panicked at 'Invalid json from get_workspaces

### ISSUE THREAD

**Issue by varac** [b12f/wlstreamer#6]
I left wlstreamer running for ~5 minutes (see https://github.com/b12f/wlstreamer/issues/2#issuecomment-730635983 how I invoked it) and suddenly it crashed:

```
...
Switched focuss= 30 q=-0.0 size=N/A time=00:00:07.66 bitrate=N/A dup=0 drop=227 speed=1.01x    
thread 'main' panicked at 'Invalid json from get_workspaces: Error("invalid type: null, expected a string", line: 49, column: 26)', src/main.rs:326:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

from dmesg:
```
[31320.883017] v4l2-loopback[1855]: opened dev:0000000020bb6870 with image:0000000000000000
[31320.883077] v4l2-loopback[1855]: opened dev:0000000020bb6870 with image:0000000000000000
[31320.883088] v4l2-loopback[960]: trying image 1366x768
[31320.883090] v4l2-loopback[1000]: s_fmt_out(0) 0...2098176
[31320.883091] v4l2-loopback[1003]: outFOURCC=YUYV
[31320.883092] v4l2-loopback[2009]: allocating 4202496 = 2101248x2
[31320.883751] v4l2-loopback[2017]: vmallocated 4202496 bytes
[31320.892805] v4l2-loopback[1994]: allocating buffers again: 4202496 4202496
[31320.988059] v4l2-loopback[1855]: opened dev:0000000020bb6870 with image:000000008b12e9ac
[31320.988084] v4l2-loopback[848]: capFOURCC=YUYV
[31320.988091] v4l2-loopback[1379]: reqbufs: 1	256=2
[31321.001272] v4l2-loopback[1855]: opened dev:0000000055b9b730 with image:0000000000000000
[31321.001286] v4l2-loopback[960]: trying image 1680x1050
[31321.001287] v4l2-loopback[1000]: s_fmt_out(0) 0...3528000
[31321.001288] v4l2-loopback[1003]: outFOURCC=YUYV
[31321.001289] v4l2-loopback[2009]: allocating 7061504 = 3530752x2
[31321.002112] v4l2-loopback[2017]: vmallocated 7061504 bytes
[31321.006695] v4l2-loopback[1994]: allocating buffers again: 7061504 7061504
```

It left a background `ffmpeg` process running which I needed to kill with `pkill ffmpeg`.

**Comment by b12f** [comment]
Hmm i guess the best would be if you could run `swaymsg -t get_workspaces` after this happens so I can see where the null comes from without having to wrap every string in an `Option`.

**Comment by varac** [comment]
ok, will do next time it happens

**Comment by b12f** [comment]
Should be fixed in 0.4.0
